composeApp/src/iosMain/kotlin/core/presentation/util/gif/Gif.ios.kt

```diff
@@ -0,0 +1,141 @@
+package core.presentation.util.gif
+
+import kotlinx.cinterop.ExperimentalForeignApi
+import kotlinx.coroutines.suspendCancellableCoroutine
+import platform.CoreFoundation.CFDataRef
+import platform.CoreGraphics.CGImageRef
+import platform.Foundation.CFBridgingRelease
+import platform.Foundation.CFBridgingRetain
+import platform.Foundation.NSData
+import platform.Foundation.NSDictionary
+import platform.Foundation.NSHTTPURLResponse
+import platform.Foundation.NSString
+import platform.Foundation.NSURL
+import platform.Foundation.NSURLRequest
+import platform.Foundation.NSURLSession
+import platform.Foundation.dataTaskWithRequest
+import platform.Foundation.valueForKey
+import platform.ImageIO.CGImageSourceCopyPropertiesAtIndex
+import platform.ImageIO.CGImageSourceCreateImageAtIndex
+import platform.ImageIO.CGImageSourceCreateWithData
+import platform.ImageIO.CGImageSourceGetCount
+import platform.ImageIO.CGImageSourceRef
+import platform.ImageIO.kCGImagePropertyGIFDelayTime
+import platform.ImageIO.kCGImagePropertyGIFDictionary
+import platform.UIKit.UIImage
+import kotlin.coroutines.resume
+
+suspend fun gifImageWithURL(url: String): UIImage? {
```

Comment:
Temporary workaround to render Gifs in iOS while Coil or Kamel doesn't add support to it.

Source code found in kotlin forum
https://slack-chats.kotlinlang.org/t/16598499/has-anyone-been-able-to-implement-a-gif-renderer-on-ios#55fc02fe-a5fe-4731-b8b8-2bb35b7c9d11